packages/desktop-client/src/components/modals/EditRuleModal.jsx

```diff
@@ -345,11 +345,11 @@ function ScheduleDescription({ id }) {
         </Text>
         <Text style={{ margin: '0 5px' }}> — </Text>
         <Text style={{ flexShrink: 0 }}>
-          <Trans>Amount</Trans>: {formatAmount(schedule._amount)}
+          <Trans>Amount:</Trans> {formatAmount(schedule._amount)}
         </Text>
         <Text style={{ margin: '0 5px' }}> — </Text>
         <Text style={{ flexShrink: 0 }}>
-          <Trans>Next</Trans>:{' '}
+          <Trans>Next:</Trans>{' '}
           {monthUtils.format(schedule.next_date, dateFormat)}
```

Comment:
It might be clearer to translators if they saw the string like `Next: {month}`, but not sure and it might not be possible